ckanext/datatablesview/blueprint.py

```diff
@@ -122,8 +122,9 @@ def ajax(resource_view_id: str):
         dtdata = {u'error': query_error}
     else:
         data = []
+        null_label = h.get_null_label()
         for row in response[u'records']:
-            record = {colname: str(row.get(colname, u''))
+            record = {colname: str(row.get(colname, u'') or null_label)
```

Comment:
this will convert any falsey value to the null_label (e.g. a false boolean column, an empty list and the number 0) use an `is None` check instead.

Reply:
Updated with `None` condition.